pythonlib/amlrealtimeai/client.py

```diff
@@ -103,3 +106,5 @@ def __predict(self, request, timeout):
                 time.sleep(sleep_delay)
                 sleep_delay = sleep_delay * 2
                 print("Retrying", rpcError)
+                # invalidate channel before retrying
+                self.__channel_usable_until = None
```

Comment:
This seems a bit convoluted - clearing usable until to count on the side effect. Why not have a method that does the channel cleanup and call it here and in _get_grpc_stub - I think that would be more robust.